vpn/src/main/java/com/duckduckgo/mobile/android/vpn/health/HealthClassifier.kt

```diff
@@ -16,26 +16,33 @@
 
 package com.duckduckgo.mobile.android.vpn.health
 
-import android.content.Context
 import com.duckduckgo.mobile.android.vpn.health.AppTPHealthMonitor.HealthState
 import com.duckduckgo.mobile.android.vpn.health.AppTPHealthMonitor.HealthState.*
-import timber.log.Timber
+import com.duckduckgo.mobile.android.vpn.model.HealthTriggerEntity
+import com.duckduckgo.mobile.android.vpn.store.AppHealthDatabase
 import javax.inject.Inject
 
-class HealthClassifier @Inject constructor(val applicationContext: Context) {
+class HealthClassifier @Inject constructor(
+    appHealthDatabase: AppHealthDatabase,
+) {
+
+    private val thresholdsDao = appHealthDatabase.appHealthTriggersDao()
```

Comment:
Should we be accessing directly a DAO rather than going through a repo or similar? :D